samples/hcls_2015.shex

```diff
@@ -77,7 +77,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     ###dcat:accessURL IRI, ### is a MAY property
     !dcat:downloadURL .,
     !void:dataDump .,
-    (void:sparqlEndpoint IRI)?,
+    void:sparqlEndpoint IRI?,
```

Comment:
Not sure what this means at all, but it works as I would expect without the brackets. ie. void:sparqlEndpoint of type IRI is optional